release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -76,6 +76,10 @@ Beginning with {product-title} {product-version}, deploying a three-node cluster
 
 For more information, see xref:../installing/installing_aws/installing-aws-three-node.adoc#installing-aws-three-node[Installing a three-node cluster on AWS], xref:../installing/installing_azure/installing-azure-three-node.adoc#installing-azure-three-node[Installing a three-node cluster on Azure], xref:../installing/installing_gcp/installing-gcp-three-node.adoc#installing-gcp-three-node[Installing a three-node cluster on GCP], and xref:../installing/installing_vsphere/installing-vsphere-three-node.adoc#installing-vsphere-three-node[Installing a three-node cluster on vSphere].
 
+[id="ocp-4-13-installation-gcp-required-permissions"]
+==== Minimum required permissions for GCP to install and delete the {product-title} cluster
+In {product-title} {product-version}, instead of using the predefined roles, you can now define your custom roles to include the minimum required permissions for Google Cloud Platform (GCP) to install and delete the {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
```

Comment:
```suggestion
In {product-title} {product-version}, instead of using the predefined roles, you can now define your custom roles to include the minimum required permissions for Google Cloud Platform (GCP) to install and delete an {product-title} cluster. These permissions are available for installer-provisioned infrastructure and user-provisioned infrastructure.
```

Reply:
Done